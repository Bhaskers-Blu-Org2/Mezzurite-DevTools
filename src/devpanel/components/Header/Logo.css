@media only screen {
  .logo {
    border-radius: 16px;
    height: 32px;
    margin-right: 8px;
    padding: 2px;
    width: 32px;
  }
}

@media only screen and (min-width: 640px) {
  .logo {
    padding: 4px;
  }
}
