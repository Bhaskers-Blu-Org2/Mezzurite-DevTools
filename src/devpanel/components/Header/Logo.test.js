import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Logo from './Logo';

describe('Logo.js', () => {
  const renderer = new ShallowRenderer();

  it('should render the logo', () => {
    const tree = renderer
      .render(<Logo />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
