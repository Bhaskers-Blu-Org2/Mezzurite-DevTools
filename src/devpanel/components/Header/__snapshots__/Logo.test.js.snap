// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo.js should render the logo 1`] = `
<svg
  className="logo"
  id="prefix__Layer_1"
  viewBox="0 0 216 216"
  x={0}
  xmlSpace="preserve"
  y={0}
>
  <style>
    .prefix__st1{fill:#f9bb10}.prefix__st2{fill:#616b77}
  </style>
  <circle
    cx={106.65}
    cy={106.71}
    fill="#363b44"
    r={73.96}
  />
  <path
    className="prefix__st1"
    d="M72.47 54.98s6.68-3.11 15.71 11.9c0 0 5.53 10.92 14.16 34.91.44 1.22 8.09 24.91 8.09 24.91s-4.69 22.21-15.79 24.73c-.58.12-18.64-56-20.21-60.57 0 0-7.07-19.22-10.41-21.42 0 0 3.14-13.55 8.45-14.46z"
  />
  <path
    className="prefix__st1"
    d="M153.9 67.96s-1.66-5.27-8.53-6.98c-4.42-1.1-9.66-1.83-14.3 1.8l-18.34 64.15s-4.9 23.16-16.06 25.38l24.09.03s11.26-.95 14.8-15.01L152 72.61s.83-4.17 1.9-4.65zM68.84 54.71l-22.69 4.85s-7.07 2.01-9.04 10.98l-19.45 85.12s-5.04 14.49-12.57 15.74l22.59 4.03s9.62-.92 16.3-22.52L61.27 68.9s2.66-11.26 7.57-14.19z"
  />
  <path
    className="prefix__st1"
    d="M141.72 58.4h17.76s16.89 1.39 21.33 13.51l18.1 64.48s1.26 7.02 12.57 12.81l-26.72 3.67s-7.4-4.43-12.17-18.72l-16.6-65.65c0 .02-1.08-9.05-14.27-10.1z"
  />
  <path
    className="prefix__st2"
    d="M15.53 160.63l6.21 1-1.17 1.73-6.01-.91zM18.05 154.02l6.81 1.1-.56 1.82-6.73-1.01zM19.57 147.34l11.55 1.88-.53 2.16-11.51-1.89zM21 141.02l6.83 1.11-.45 1.93-6.8-1.12zM22.55 134.28l6.97 1.04-.47 1.92-6.93-1.04zM24.07 127.63l11.55 1.94-.43 2.12-11.6-1.92zM25.55 121.17l6.83 1.1-.45 1.94-6.82-1.13zM27.07 114.52l6.96 1.04-.47 1.92-6.93-1.04zM28.6 107.8l11.64 1.48-.43 2.1-11.7-1.44zM30.07 101.38l6.89.83-.45 1.95-6.88-.86zM31.51 95.02l7.03.77-.47 1.96-6.99-.82zM33.16 87.85l11.64 1.48-.43 2.1-11.7-1.44zM34.79 80.58l6.89.84-.45 1.97-6.85-.89zM36.47 73.41l7.09.52-.5 2.04-7.03-.64z"
  />
</svg>
`;
